Compilation fails on a project using RN 0.48.x

Here is a reproduction repository: https://github.com/exKAZUu/react-native-kontaktio-issue20
`react-native run-ios` shows the following errors.

```
In file included from /Users/exkazuu/Projects/AwesomeProject/node_modules/react-native-kontaktio/ios/KontaktBeacons.m:1:
In file included from /Users/exkazuu/Projects/AwesomeProject/node_modules/react-native-kontaktio/ios/KontaktBeacons.h:8:
In file included from ../../react-native/React/Modules/RCTEventEmitter.h:10:
In file included from /Users/exkazuu/Projects/AwesomeProject/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTBridge.h:13:
/Users/exkazuu/Projects/AwesomeProject/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTBridgeModule.h:54:16: error: redefinition of 'RCTMethodInfo'
typedef struct RCTMethodInfo {
               ^
In file included from /Users/exkazuu/Projects/AwesomeProject/node_modules/react-native-kontaktio/ios/KontaktBeacons.m:1:
In file included from /Users/exkazuu/Projects/AwesomeProject/node_modules/react-native-kontaktio/ios/KontaktBeacons.h:2:
../../react-native/React/Base/RCTBridgeModule.h:54:16: note: previous definition is here
typedef struct RCTMethodInfo {
               ^
In file included from /Users/exkazuu/Projects/AwesomeProject/node_modules/react-native-kontaktio/ios/KontaktBeacons.m:1:
In file included from /Users/exkazuu/Projects/AwesomeProject/node_modules/react-native-kontaktio/ios/KontaktBeacons.h:8:
In file included from ../../react-native/React/Modules/RCTEventEmitter.h:10:
In file included from /Users/exkazuu/Projects/AwesomeProject/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTBridge.h:13:
/Users/exkazuu/Projects/AwesomeProject/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTBridgeModule.h:58:3: error: typedef redefinition with different types ('struct (anonymous struct at /Users/exkazuu/Projects/AwesomeProject/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTBridgeModule.h:54:16)' vs 'struct RCTMethodInfo')
} RCTMethodInfo;
  ^
In file included from /Users/exkazuu/Projects/AwesomeProject/node_modules/react-native-kontaktio/ios/KontaktBeacons.m:1:
In file included from /Users/exkazuu/Projects/AwesomeProject/node_modules/react-native-kontaktio/ios/KontaktBeacons.h:2:
../../react-native/React/Base/RCTBridgeModule.h:58:3: note: previous definition is here
} RCTMethodInfo;
  ^
In file included from /Users/exkazuu/Projects/AwesomeProject/node_modules/react-native-kontaktio/ios/KontaktBeacons.m:1:
In file included from /Users/exkazuu/Projects/AwesomeProject/node_modules/react-native-kontaktio/ios/KontaktBeacons.h:8:
In file included from ../../react-native/React/Modules/RCTEventEmitter.h:10:
In file included from /Users/exkazuu/Projects/AwesomeProject/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTBridge.h:13:
/Users/exkazuu/Projects/AwesomeProject/ios/build/Build/Products/Debug-iphonesimulator/include/React/RCTBridgeModule.h:65:11: warning: duplicate protocol definition of 'RCTBridgeModule' is ignored [-Wduplicate-protocol]
@protocol RCTBridgeModule <NSObject>
          ^
In file included from /Users/exkazuu/Projects/AwesomeProject/node_modules/react-native-kontaktio/ios/KontaktBeacons.m:1:
In file included from /Users/exkazuu/Projects/AwesomeProject/node_modules/react-native-kontaktio/ios/KontaktBeacons.h:2:
../../react-native/React/Base/RCTBridgeModule.h:65:11: note: previous definition is here
@protocol RCTBridgeModule <NSObject>
          ^
1 warning and 2 errors generated.



** BUILD FAILED **
```

--- a/ios/KontaktBeacons.h
+++ b/ios/KontaktBeacons.h
@@ -1,13 +1,13 @@
-#if __has_include("RCTBridgeModule.h")
-  #import "RCTBridgeModule.h"
-#else
+#if __has_include(<React/RCTBridgeModule.h>)
   #import <React/RCTBridgeModule.h>
+#else
+  #import "RCTBridgeModule.h"
 #endif
 
-#if __has_include("RCTEventEmitter.h")
-  #import "RCTEventEmitter.h"
-#else
+#if __has_include(<React/RCTEventEmitter.h>)
   #import <React/RCTEventEmitter.h>
+#else
+  #import "RCTEventEmitter.h"
 #endif
 
 

